subsys/logging/Kconfig

```diff
@@ -249,6 +249,91 @@ config LOG_BACKEND_UART_FORMAT_TIMESTAMP
 	default y
 	help
 	  When enabled timestamp is formatted to hh:mm:ss:ms,us.
+	  
+config LOG_FRONTEND
+	bool "Enable GPIO frontend"
+	help
+	  When enabled frontend is using GPIO pins to output logs.
+	  
+choice
+	prompt "Pin numbers choice"
+
+config LOG_FRONTEND_PINS_IN_SEQUENCE
+	bool "Choose pin number for only first bit of the message"
+	depends on LOG_FRONTEND
+
+config LOG_FRONTEND_PINS_NOT_IN_SEQUENCE
+	bool "Choose pin number for every bit of the message"
+	depends on LOG_FRONTEND
+
+endchoice
+
+config LOG_FRONTEND_PORT_NUMBER
+	int "Choose port number"
+	depends on LOG_FRONTEND
+	default 1
+	range 0 1
+
+config LOG_FRONTEND_PIN_CLK
+	int "Choose pin number for clock signal"
+	depends on LOG_FRONTEND
+	default 10
+	range 0 31
+
+config LOG_FRONTEND_PINS_IN_SEQUENCE_FIRST_BIT
+	int "Pin number for first bit"
+	depends on LOG_FRONTEND_PINS_IN_SEQUENCE
+	default 1
+	range 0 31
```

Comment:
You shall limit this range to: `max val - 9`

Please take into account that SOC_NRF52840_QIAA has PIN range 0-47